internal/telemetry/segment.go

```diff
@@ -154,3 +154,15 @@ func timeFromUnixTimestamp(timestamp string) (time.Time, error) {
 func UnixTimestampFromTime(t time.Time) string {
 	return strconv.FormatInt(t.Unix(), 10)
 }
+
+func shellAccess() shellAccessKind {
+	// Check if running in devbox cloud
+	if os.Getenv("DEVBOX_REGION") != "" {
+		// Check if running via ssh tty (i.e. ssh shell)
+		if os.Getenv("SSH_TTY") != "" {
```

Comment:
@loreto I think this approach is closer to what you wanted where we test for `ssh` versus `browser` based on access methodology.

Reply:
Sounds good! Just make sure you test it  :)

Reply:
I verified that SSH_TTY isn't present in browser shell: 
https://github.com/jetpack-io/devbox/pull/562#discussion_r1094983619